Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -57,22 +57,13 @@ the CRs to their prior state without even necessarily suffering cluster downtime
     kubectl -n rook-ceph get configmap -o yaml > configmaps.yaml
     ```
 
-3.  Back up and remove the `ValidatingWebhookConfiguration`. This is the resource which connects Rook custom resources to the
-    operator pod's validating webhook. Because the operator is unavailable, we must temporarily disable the valdiating webhook in
-    order to make changes.
+3.  Disable the `ValidatingWebhookConfiguration`. This is the resource which connects Rook custom resources
+    to the operator pod's validating webhook. Because the operator is unavailable, we must temporarily disable
+    the valdiating webhook in order to make changes.
 
-    1.  Save the `ValidatingWebhookConfiguration` responsible for controlling Rook resources. (These are not namespaced
-        resources.)
-
-        ```console
-        kubectl get ValidatingWebhookConfiguration rook-ceph-webhook -o yaml > rook-ceph-webhook.yaml
-        ```
-
-    2.  Delete the `ValidatingWebhookConfiguration`.
-
-        ```console
-        kubectl delete ValidatingWebhookConfiguration rook-ceph-webhook
-        ```
+    ```console
+    kubectl -n rook-ceph patch deployments.apps rook-ceph-operator -p '{"spec": {"template": {"spec": {"containers": [{"name": "rook-ceph-operator", "env":[{"name":"ROOK_DISABLE_ADMISSION_CONTROLLER","value":"true"}]}]}}}}'
```

Comment:
Does this step need to be first so the operator can disable it before it's scaled down? In step 1 (line 46), the operator was scaled down.

Reply:
If we kept the step to delete the validating webhook configuration, would that be sufficient to disable it? I just wonder if that's simpler than disabling the setting in the operator or if there are multiple steps the operator needs to handle internally to keep it simple. If we didn't change the operator setting, then no need for step 8 below and the operator could just repair the webhook automatically when it's scaled up.

Reply:
> Does this step need to be first

right, it should be first step

Reply:
> If we kept the step to delete the validating webhook configuration, would that be sufficient to disable it? 

yes, it should be enough, and this is one of the things we do when webhook is disabled

>I just wonder if that's simpler than disabling the setting in the operator or if there are multiple steps the operator needs to handle internally to keep it simple. If we didn't change the operator setting, then no need for step 8 below and the operator could just repair the webhook automatically when it's scaled up.

yes, I think if we just delete the validating webhook and not changing the operator env will do the work.

Reply:
> Does this step need to be first so the operator can disable it before it's scaled down? In step 1 (line 46), the operator was scaled down.